From a3a1bb14b60c215bd39c5ffde16ffe85fcafe109 Mon Sep 17 00:00:00 2001
From: Martin <zach@b1-systems.de>
Date: Sun, 15 May 2022 00:51:59 +0200
Subject: [PATCH]  hwdb: Add accel orientation for the I15-TG

Add accel orientation for the Cube iWork 10 I15-TG

(cherry picked from commit c4c336e0cdf66a4c0922f2b77a7e6e3811287114)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index d883db22a1..3fb308f3f1 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -277,6 +277,7 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnCube:pni18B:*
 
 # Cube iWork 10 Flagship
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnCube:pnI15-TC:*
+sensor:modalias:acpi:BOSC0200*:dmi:*:svnALLDOCUBE:pnI15-TG:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
 # Cube iWork 11 Stylus
